gatorgrade/input/parse_config_function.py

```diff
@@ -0,0 +1,11 @@
+"""Returns the list of commands to be run through gatorgrader"""
+from command_line_generator import generate_checks # Import function to generate shell and gatorgrader checks
+from in_file_path import parse_yaml_file, reformat_yml_data, add_checks_to_list # Import functions to parse and set up yaml file 
+
+def parse_config(file):
+    """Parse the input yaml file and generate specified checks
+    
+    Args:
+        file: Yaml file containing gatorgrade and shell command checks 
+    """
+    generate_checks(reformat_yaml_data(parse_yaml_file(file))) # Call previously generated function to modify file
```

Comment:
Hey Michael, I think the parse_config function needs to return a value. Moreover, the generate_checks function returns a dictionary, thus you need to assign the output to a variable or just return the output in the parse_config function.

Reply:
I had discussed it with Professor Heinert and we ascertained that a return value wasn't necessary, nor was the need to assign the output to a variable based on the next steps of the project, which doesn't involve our team

Reply:
Alright, thanks for letting me know.

Reply:
@MAbraham24, actually, @tuduun is correct! We must have overlooked the need for a return value during our discussion.

Reply:
Understood, will implement that!